site/topics/functions/functions.rst

```diff
@@ -237,42 +237,54 @@ Abstraction
     * For example, when driving a car, do you think about the pistons firing?
 
 
-Back to concrete things...
-==========================
+Format of a Function
+====================
 
-* The general format for defining a function is::
+* The general format for defining a function is as follows
```

Comment:
[Not quite](https://docs.python.org/3/reference/grammar.html). This is actually quite specific. Yeet that _general_.